### ISSUE TITLE
Partially broken high-contrast support in Poedit

### ISSUE THREAD

**Issue by k-kolev1985** [vslavik/poedit#416]
From v1.7 I think, Poedit's UI started having problems supporting the high-contrast color mode in Windows. Even though my OS is set to use the high-contrast black color theme in Windows 10, some controls in Poedit's UI refuses to obey the colors defined in Windows and uses his own ones. Version 1.6 of Poedit does not have such issues. I'll attach a few screenshots. The first one is from Poedit v1.6 and the other two - from Poedit v2.0.2.

Steps to reproduce it:
1. Install Poedit v2.0.2.
2. Go to Windows Settings -> Ease of Access -> High Contrast and from the "High-Contrast Theme" combo box choose "High-Contrast Black". Then click "Apply".
3. Open a .po file in Poedit v2.0.2 and look around the UI of Poedit.

Expected results:
Poedit should use the colors defined in the selected Windows theme for all of its components. In this case - black background and white text.

Actual results:
Poedit uses white background (and in some cases - black text) for some of its controls.

Test environment:
- Operating system: Windows 10 Pro version 1703 (build 15063.483), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- Poedit version: 2.0.2, in english.

Can this be fixed?

Thanks much in advance!
![screenshot_2017-07-22_12-20-22](https://user-images.githubusercontent.com/4395934/28490209-0242b4fe-6edd-11e7-933f-027b793311cf.png)
![screenshot_2017-07-22_12-22-15](https://user-images.githubusercontent.com/4395934/28490215-173e4d5a-6edd-11e7-90a4-d9bd080ed910.png)
![screenshot_2017-07-22_12-22-27](https://user-images.githubusercontent.com/4395934/28490218-1c8ac4be-6edd-11e7-94a3-4761b951caaa.png)

**Comment by vslavik** [comment]
I'm aware. See also #273, which is the same thing on Linux. I hoped for a proper fix with dynamically reacting to settings changes, but I suppose static support is better than none.

**Comment by k-kolev1985** [comment]
So, is it fixable or not?

**Comment by vslavik** [comment]
Did I say anything about not being able to fix it? Did I close the bug as WONTFIX? There's your answer.

**Comment by k-kolev1985** [comment]
OK, sorry for the misunderstanding.

**Comment by k-kolev1985** [comment]
The problem seams to be almost completely fixed. The only incorrect color now is the color of the text for a selected item in the translations list. See attached screenshot - in it, the correct color should be black, as it is for the highlighted menu item, but in the translations list the text for a highlighted item is white, making it hard to read.

**Comment by k-kolev1985** [comment]
Oops, forgot the attachment. Here it is.
![2019-01-02_18-36-45](https://user-images.githubusercontent.com/4395934/50602530-3491f080-0ec0-11e9-996f-c6c3caaaaeb9.jpg)
